plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -9501,8 +9503,8 @@ private List<TestingHiveStorageFormat> getAllTestingHiveStorageFormat()
                 // CSV supports only unbounded VARCHAR type
                 continue;
             }
-            if (hiveStorageFormat == REGEX) {
-                // REGEX format is read-only
+            if (hiveStorageFormat == REGEX || hiveStorageFormat == ESRI) {
```

Comment:
Nit: probably break this out into a separate if statement to keep these cleaner over time